docs/README.md

```diff
@@ -1,22 +1,34 @@
-Kyma is an application runtime that provides you a flexible and easy way to connect, extend, and customize your applications in the cloud-native world of Kubernetes.
+## What is Kyma
+
+Kyma is an opinionated set of Kubernetes-based modular building blocks, including all necessary capabilities to develop and run enterprise-grade cloud-native applications.
+It is the open path to the SAP ecosystem supporting business scenarios end-to-end.
+
+Kyma is an actively maintained open-source project supported by SAP. The Kyma project is also a foundation of Kubernetes Runtime which is a part of SAP Business Technology Platform (BTP). You can use Kyma modules in your own Kubernetes cluster, or try the managed version from SAP BTP with a ready-to-use Kubernetes cluster powered by Gardener. 
 
 ![overview](./assets/kyma-overview.svg)
 
-Out of the box, Kyma offers various functionalities, such as:  
+## Modular Kyma
+
+Classic Kyma offered a fixed set of preconfigured components whose development rhythm is synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. Each module has its own custom resource that holds the desired configuration and the operator that reconciles the configuration. Kyma project is currently in the transition phase. Some components are already independent modules, but others are still part of the big Kyma release and are installed with `kyma deploy` command. With each successive release, fewer components will be available within the preconfigured Kyma, but more and more will be offered as independent modules.
```

Comment:
I would also link to the `docs/frequently-asked-questions.md` from the overview page, what do you think?

Reply:
> I would also link to the `docs/frequently-asked-questions.md` from the overview page, what do you think?

My intention was to replace Advantages section but was blurry. I chose the FAQ format as more direct and easier to read  (IMHO). I wanted to keep it on that page as the page is relatively short after editing.